### PR TITLE
Add synapse.tools.genpkg docs

### PR DESCRIPTION
**Pull request by Cisphyx**

### CODE REVIEW

File: docs/synapse/userguides/syn_tools_genpkg.ipynb [hunk 1]
```diff
@@ -0,0 +1,242 @@
...
+   "metadata": {},
+   "source": [
+    ".. highlight:: none\n",
+    "\n",
+    ".. _syn-tools-genpkg:\n",
+    "\n",
+    "Synapse Tools - genpkg\n",
+    "========================\n",
+    "\n",
+    "The Synapse ``genpkg`` tool can be used to generate a Storm :ref:`gloss-package` containing new Storm commands and Storm modules from a YAML prototype and optionally push it to a Cortex or PkgRepo. \n",
```
Comment (vEpiphyte) [inline docs/synapse/userguides/syn_tools_genpkg.ipynb:14]: prototype is a weird word

File: docs/synapse/userguides/syn_tools_genpkg.ipynb [hunk 2]
```diff
@@ -0,0 +1,242 @@
...
+    "    foopkg.yml\n",
+    "    storm/\n",
+    "    ├── commands/\n",
+    "    │   └── foocmd\n",
+    "    ├── modules/\n",
+    "    │   └── foomod\n",
+    "    └── optic/\n",
+    "        └── index.html\n",
+    "\n",
+    "Commands and modules defined in the package YAML file are expected to have corresponding files containing the Storm code for their implementation. It is not required to have both commands and modules in a Storm package, you may have a package with only commands, or only modules.\n",
```
Comment (jnwatson) [inline docs/synapse/userguides/syn_tools_genpkg.ipynb:55]: "package, you" -> "package; you"

File: docs/synapse/userguides/syn_tools_genpkg.ipynb (hunk 1 above)
Comment (jnwatson) [reply docs/synapse/userguides/syn_tools_genpkg.ipynb:14]: specification would be a good alternative.

Comment (invisig0th) [reply docs/synapse/userguides/syn_tools_genpkg.ipynb:14]: `definition` would be best.

**Review by vEpiphyte — APPROVED** [review]

**Pull request merged**